network-api/networkapi/wagtailpages/templates/wagtailpages/blocks/annotated_image_block.html

```diff
@@ -2,29 +2,19 @@
 
 {% image value.image original as img %}
 
-{% if value.wide_image %}
-    </div> <!-- End .blog-body -->
-  </div> <!-- End .cms -->
-</div> <!-- End .col-lg-8 -->
-
-<div class="col-lg-12">
-{% endif %}
-
-  <figure class="my-default">
-    <img src="{{ img.url }}" alt="{{ value.altText }}" class="w-100" />
-    {% if value.caption %}
-    <figcaption class="body-small d-block mt-2">
-      {% if value.captionURL %}<a href="{{ value.captionURL }}">{% endif %}
-      {{ value.caption }}
-      {% if value.captionURL %}</a>{% endif %}
-    </figcaption>
-    {% endif %}
-  </figure>
-
-{% if value.wide_image %}
+<div class="container {{block.block_type}}-block {% if value.wide_image %} wide mw-100 {% endif %}">
```

Comment:
Starting its own container, and if the image is wide, we are going to set it to max-width 100.
Then in the 2nd next div, we are saying if the image is wide, make it span across the screen, if not, use col-8.
The rest is from the original template